orbit/pkg/table/extension_darwin.go

```diff
@@ -59,5 +60,9 @@ func PlatformTables() []osquery.OsqueryPlugin {
 		// osquery version 5.5.0 and up ships a unified_log table in core
 		// we are renaming the one from the macadmins extension to avoid collision
 		table.NewPlugin("macadmins_unified_log", unifiedlog.UnifiedLogColumns(), unifiedlog.UnifiedLogGenerate),
+
+		// Kolide tables.
+		// kolide_firmwarepasswd table. Only returns valid data on a Mac with an Intel processor. Background: https://support.apple.com/en-us/HT204455
+		firmwarepasswd.TablePlugin(serverClient, kolideLogger),
```

Comment:
Seems `serverClient` is unused and so we should pass a `nil`? (Maybe other extensions make use of it and this was copy/pasted when implementing new tables.)